tests/test_sensor.py

```
@@ -2693,6 +2754,429 @@ def add_entities(entities: Iterable[Any], _update_before_add: bool = False) -> N
+    config_entry = setup_sensor_reconciliation_entry(
+        make_config_entry,
+        coordinator_data={
+            "openvpn": {"clients": {"bad": None}, "servers": {}},
+            "wireguard": {"clients": {}, "servers": {}},
+        },
+        sync_vpn=True,
+    )
+    captured = capture_reconciled_desired_entities(monkeypatch)
+
+    await async_setup_entry(
+        MagicMock(), config_entry, cast("AddEntitiesCallback", lambda entities, _=False: None)
```

Ruff (0.15.21)

[warning] Unused lambda argument: `entities`

(ARG005)